src/routes/PrimaryLayout/PrimaryLayout.js

```diff
@@ -248,15 +272,18 @@ const createCommunityRoutes = [
 ]
 
 const redirectRoutes = [
-  { from: '/tag/:topicName', to: '/all/topics/:topicName' },
+  { from: '/tag/:topicName', to: '/all/:topicName' },
   { from: '/c/:slug/tag/:topicName', to: '/c/:slug/:topicName' },
   { from: '/c/:slug/join/:accessCode/tag/:topicName', to: '/c/:slug/join/:accessCode/:topicName' },
   { from: '/p/:postId', to: '/all/p/:postId' },
   { from: '/u/:personId', to: '/m/:personId' },
   { from: '/c/:slug/about', to: '/c/:slug' },
-  { from: '/c/:slug/people', to: '/c/:slug/members' },
-  { from: '/c/:slug/invite', to: '/c/:slug/settings/invite' },
-  { from: '/c/:slug/events', to: '/c/:slug' }
+  { from: '/c/:slug/people', to: '/c/:slug/:view(members)' },
+  { from: '/c/:slug/invite', to: '/c/:slug/:view(settings)/invite' },
+  { from: '/c/:slug/events', to: '/c/:slug' },
+  // redirects for context switching into global contexts
+  { from: '/all/members', to: '/all' },
+  { from: '/public/(members|topics)', to: '/public' }
```

Comment:
This looked strange to me. No param key needed, I get it, and I'm sure it works as you tested it. It does right?

Reply:
yep! Seems to land at the intended destinations, and I figured having explicit redirects mapped out there would prevent any future developers from encountering surprises